src/js/hooks/list-pages/outbound/useOutboundListTableData.jsx

```diff
@@ -36,6 +43,7 @@ const useOutboundListTableData = (filterParams) => {
       requestedBy: requestedBy?.id,
       createdBy: createdBy?.id,
       updatedBy: updatedBy?.id,
+      shipmentType: shipmentType?.map?.(({ id }) => id),
```

Comment:
Yes, I am sure it is working - it was not working, because you were doing it like this:
```javascript
 shipmentType?.map(({ id }) => id),
```
as `.map` is a function, it also needs a `?` after calling `map`, so it doesn't get executed if anything before that is `undefined` or `null`

Reply:
I love that explanation, thanks 😁